tests/test_conf.py

```diff
@@ -73,6 +83,8 @@
     "version_files": ["commitizen/__version__.py", "pyproject.toml"],
     "style": [["pointer", "reverse"], ["question", "underline"]],
     "changelog_file": "CHANGELOG.md",
+    "hooks_pre_bump": ["scripts/generate_documentation.sh"],
+    "hooks_post_bump": ["scripts/slack_notification.sh"],
```

Comment:
I think it would be better for us to add at least one run-able test

Reply:
@Lee-W I fully agree, but I'm not a real python dev myself and am not familiar with the pytest framework. If it wouldn't be too much work, could you perhaps pitch in here?

Reply:
yep, I'll take a deeper look later and see how I can help on this :)

Reply:
just added a test case to verify it